rust_src/src/keymap.rs

```diff
@@ -2,9 +2,8 @@
 
 use remacs_macros::lisp_fn;
 use remacs_sys::{current_global_map as _current_global_map, globals, EmacsInt, CHAR_META};
-use remacs_sys::{access_keymap, get_keymap, maybe_quit, Fevent_convert_list, Ffset, Fpurecopy,
-                 Fset};
-use remacs_sys::Qkeymap;
+use remacs_sys::{access_keymap, get_keymap, maybe_quit, Fcons, Fevent_convert_list, Ffset,
+                 Fmake_char_table, Fpurecopy, Fset, Qkeymap, Qnil};
```

Comment:
Ok, this is another remacs style item.

If we merge them all into one import statement the merge conflicts are a constant pain. So what we do is separate them by kind. All of the Qfoo constants in one import. All of the Ffunc function imports in another. The types like EmacsInt and LispObject in another. Then everything else.

Reply:
Aha. That makes sense. Is this written somewhere in the docs? If yes, then sorry if I missed it. If not, then might be a good idea to add it.

Alright, I'll fix that later and update.

Reply:
No, it is not. I thought it was documented. Thanks! Off to update the wiki...

Reply:
Ah, nevermind. Yes, it is in the CONTRIBUTING.md

Reply:
I added a link to the file from the wiki porting page.